Minor update to accept/publish handoff between editors and EiCs (#694)
openjournals/joss: docs/editing.md

- Check the archive deposit has the correct metadata (title and author list), and request the author edit it if it doesn’t match the paper.
- Run `@whedon set <doi> as archive`.
- Run `@whedon set <v1.x.x> as version` if the version was updated.

> - Run `@whedon accept` to generate the final proofs and have Whedon notify the `@openjournals/joss-eics` team that the paper is ready for final processing.

Reviewer: instead of "and have Whedon notify..." perhaps it should be ", which has Whedon notifiy..."

> - Run `@whedon accept` to generate the final proofs, which has Whedon notify the `@openjournals/joss-eics` team that the paper is ready for final processing.

Reviewer 2: ```suggestion
- Run `@whedon accept` to generate the final proofs and notify the `@openjournals/joss-eics` team that the paper is ready for final processing.
```
